config/default.yml

```diff
@@ -18,5 +18,9 @@ accounts:
 
 public:
   authenticationMechanism: 'EMAIL' #or 'PHONE'
+  default_otp:
+    enabled: false
+    code: '1234'
```

Comment:
I would expect `code` and `whitelist` are not needed since `enabled` is false, remove `code` and `whitelist`